.github/workflows/ci.yml

```diff
@@ -39,17 +39,17 @@ jobs:
       - run: npm ci
       - run: npm run codegen:apollo
       - run: npm run lint
-      # - run: npm test
+      - run: npm test
```

Comment:
`codegen:apollo` & `lint` would run twice with this as `npm test` runs them all already. So we should be able to reduce this to:

```yml
- run: npm ci
- run: npm test
```